packages/components/src/internal/components/domainproperties/models.tsx

```diff
@@ -1700,6 +1719,8 @@ export class DomainException
 
         allFieldErrors = allFieldErrors.map(error => {
             const indices = domain.fields.reduce((indexList, field, idx, iter): List<number> => {
+                return indexList;
```

Comment:
@labkey-jeckels I'm a little surprised by this change? Was this move of the `return indexList` intentional or did the linting command make this change?

Reply:
I don't know where it came from but it definitely is wrong. I didn't intend to make the change. I reverted that change and re-ran linting and it wasn't reintroduced, which is good.